### PR TITLE
feat: add auto_comfort service

### PR DESCRIPTION
**Pull request by barlock**
In support of #33 

This adds a service for smart mode features.

Depends on: https://github.com/mikelawrence/aiosenseme/pull/11

I'm creating this PR as a draft as I need to refactor it on ☝️ once it's merged. I figured I'd create it now so the whole thing could be used while getting this merged.

### CODE REVIEW

File: custom_components/senseme/aiosenseme/__init__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import asyncio
```
Comment (barlock) [inline custom_components/senseme/aiosenseme/__init__.py:1]: This file will be removed once https://github.com/mikelawrence/aiosenseme/pull/11 is merged

File: custom_components/senseme/aiosenseme/device.py [hunk 2]
```diff
@@ -0,0 +1,1266 @@
+"""SensemeFan Class.
```
Comment (barlock) [inline custom_components/senseme/aiosenseme/device.py:1]: This file will be removed once https://github.com/mikelawrence/aiosenseme/pull/11 is merged

File: custom_components/senseme/__init__.py [hunk 3]
```diff
@@ -2,9 +2,9 @@
 import asyncio
 import logging
 
-from aiosenseme import SensemeDevice
+from .aiosenseme.device import SensemeDevice
```
Comment (barlock) [inline custom_components/senseme/__init__.py:5]: These changes will be removed once mikelawrence/aiosenseme#11 is merged

Comment (mikelawrence) [reply custom_components/senseme/__init__.py:5]: You can install local copy of the aiosenseme library and you won't have to make changes like those in the `__init__.py` file.
```
pip install -e directory_of_aiosenseme_clone
```

**Comment by mikelawrence** [comment]
This integration is currently being added to Home Assistant.

**Pull request closed without merging**